Newspost regarding Official Extensions removal (#1112)
tachiyomiorg/website: website/src/news/2024-01-09-extensions-removal.md

---
type: article
title: Extensions list removed

> description: Due to extenuating circumstances and legal considerations.

Reviewer: ```suggestion
description: Due to extenuating circumstances.
```

date: "2024-01-09"
---

# Extensions list removed

**Dear Tachiyomi Community,**

> Due to extenuating circumstances and legal considerations, we have made the decision to purge our list of extensions to ensure the long-term sustainability of **Tachiyomi**.

Reviewer: ```suggestion
Due to extenuating circumstances, we have made the decision to purge our list of extensions to ensure the long-term sustainability of **Tachiyomi**.
```

As of now, **Tachiyomi** is transitioning to a fully bring-your-own-content model. What this means is that you can still enjoy **Tachiyomi** for manga reading, but you'll need to source and add your own content.

To make this transition smoother for our users, we are retaining a select few extensions that support connecting to personal "Plex-like" systems, such as **Komga**.

> For the complete list of available extensions, please visit [extensions](/extensions/).

We strongly encourage all users to exercise caution, and avoid utilizing extensions from unknown repositories, as they may contain malware.

> It's important to note that existing extensions will continue to work, but we cannot guarantee their functionality indefinitely.

Reviewer: ```suggestion
It's important to note that while existing extensions may continue to work, but we cannot guarantee their functionality indefinitely.
```
